adapter/internal/discovery/xds/server.go

```diff
@@ -345,6 +357,30 @@ func UpdateAPI(vHost string, apiProject model.ProjectAPI, environments []string)
 		return nil, validationErr
 	}
 
+	// create client map for API
+	var clientCerts []model.Certificate
+	if len(apiProject.ClientCerts) > 0 && len(apiProject.DownstreamCerts) > 0 {
+		for _, certFile := range apiProject.ClientCerts {
+			var certificate model.Certificate
+			if certBytes, found := apiProject.DownstreamCerts[certFile.CertificateName]; found {
+				certificate.Alias = certFile.Alias
+				certificate.Tier = certFile.Tier
+				certificate.Content = certBytes
+				clientCerts = append(clientCerts, certificate)
+				delete(apiProject.DownstreamCerts, certFile.CertificateName)
+			} else {
+				logger.LoggerXds.ErrorC(logging.ErrorDetails{
+					Message: fmt.Sprintf("Certificate file %v not found for the alias %v in the API %s:%s", certFile.CertificateName,
+						certFile.Alias, apiYaml.Name, apiYaml.Version),
+					Severity:  logging.MINOR,
```

Comment:
Shall we use `apiYaml.ID` instead of name and version. (or else we could add it as an additional string too). If there is not ID (this could happen in standalone projects), we should print apiName and version anyway

Reply:
Included.